common/shlibs

```diff
@@ -4017,3 +4017,4 @@ libmt32emu.so.2 libmt32emu-2.5.1_1
 libqrtr-glib.so.0 libqrtr-glib-1.0.0_1
 libstemmer.so.2 libstemmer-2.1.0_1
 libdecor-0.so.0 libdecor-0.1.0_1
+libfontmanager.so.0 fontmanager-0.8.7_1
```

Comment:
Remove until other package uses library, libfontmanager should be split at that time.